### DISCUSSION TITLE
SEI-on-cracks during calendar aging

### DISCUSSION THREAD

**Discussion by fsenoner** [pybamm-team/PyBaMM#3577] [Q&A]
Good morning dear PyBaMM team (@DrSOKane , @brosaplanella , @tinosulzer and co.),

I have a question regarding the SEI-on-cracks submodel. I am simulating calendaric aging with a check-up every 50 days. There are two models (DFN) with the following settings:
"SEI on cracks": "false"
"SEI porosity change": "false",
"SEI": "EC reaction limited".

and 

"SEI on cracks": "true"
"SEI porosity change": "false",
"SEI": "EC reaction limited".

I did not enable the particle mechanics submodel in this simulation. This means that the area on which the SEI-on-cracks can grow remains the same throughout the simulation. I plotted the "Loss of capacity to SEI [A.h]" and the "Loss of capacity to SEI on cracks [A.h]".
 
![image](https://github.com/pybamm-team/PyBaMM/assets/130640867/62e6635e-9144-486f-8932-10f908809441)

The blue line shows the simulation with "SEI on cracks" disabled and the orange is the one with "SEI on craks" enabled.

These two plots show the same behavior, but the loss of capacity to SEI on cracks is approximately double the loss of capacity to SEI. This makes no sense to me because the two SEI growth models should use the same exchange current density. For better understanding, I also plotted the "X-averaged outer SEI thickness [m]" and the "X-averaged outer SEI on cracks thickness [m]". These two plots show exactly the same values. On the right you can see the "X-averaged negative particle crack length". This plot is zero for the model with "SEI on cracks" disabled and constant ("Negative electrode initial crack length") for the model with sei on cracks enabled.

![image](https://github.com/pybamm-team/PyBaMM/assets/130640867/57777e46-4735-4f01-8b8c-bb5a42f8ab82)

Could you please explain why there is a difference in the loss of capacity due to SEI and SEI on cracks, even though both SEI layers have the same thickness?

Thank you
Fabian Senoner

**Comment by fsenoner** [comment]
I found that in the file base_sei.py, there is a factor called "roughness," which is calculated as follows:
![image](https://github.com/pybamm-team/PyBaMM/assets/130640867/366f7c19-bfba-45bf-9579-f6a17d332260)

When calculating the loss of capacity to SEI on cracks, the thickness is multiplied by roughness -1, along with other variables.

If I calculate the roughness in this specific case, I get (initial crack length, crack density and crack width taken from OKane parameter set)
![image](https://github.com/pybamm-team/PyBaMM/assets/130640867/222e39c5-f51e-4ef8-93b6-4daef3c103e7)
0.61 Ah *1.908 = 1.16 Ah (=  "Loss of capacity to SEI on cracks")

This factor represents exactly the difference between the "Loss of capacity to SEI" and the "Loss of capacity to SEI on cracks". This is clear to me. Could you please explain why the roughness factor is used and the physical background of the formula for the roughness factor?

Thank you

Fabian

**Comment by DrSOKane** [comment]
Hi Fabian, if "SEI on cracks" is enabled, particle mechanics is automatically enabled. SEI on cracks that do not propagate is something I'm working on but haven't put into PyBaMM yet.
